### PR TITLE
Global CSS styles for table components 

### PR DESCRIPTION
**Pull request by GracielaCesareo**
This PR adds global CSS styles to table components. Also, status label styles.
Rules for styles are specified on Figma file [here](https://www.figma.com/file/EOjhveLSKu45LroeUrMCeD/Ombulabs?node-id=3%3A4145)

Tests

Browsers
	* Chrome
	* Safari
	* Firefox

Devices
	* Desktop

Screen Shot 2021-04-07 at 16 03 55

### CODE REVIEW

File: vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_tables.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
+table{
+        width: 100%;
```
Comment (FionaDL) [inline vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_tables.scss]: There are some things going on with the spacing in this section. Could you fix that? :)

**Comment by GracielaCesareo** [comment]
@FionaDL Fixes done

**Review by FionaDL — APPROVED** [review]
@GracielaCesareo looks good to me. @gunnarthor or @cleicar this is ready for QA.

File: index.html [hunk 2]
```diff
@@ -145,6 +148,75 @@ <h2>Icons</h2>
...
+                    <a href="#"><i class="fas fa-redo-alt"></i></a>
+                    <a href="#"><i class="fas fa-trash"></i></a>
+                  </div>
+                </td>
+              </tr>
+              
+              <tr class="">
+                <td class="number-cell">099</td>
+                <td>Lorem ipsum dolor sit amet consectetur, adipiscing elit habitasse orci ut, dapibus nulla ante ultrices. Dignissim dictum ad habitasse curabitur quis praesent, facilisi magna curae lectus euismod primis volutpat, congue semper erat suscipit imperdiet</td>
+                <td><span class="status-label pending">Peding</span></td>
```
Comment (cleicar) [inline index.html]:
```suggestion
                <td><span class="status-label pending">Pending</span></td>
```

**Review by cleicar — APPROVED** [review]
Looks good to go!

**Pull request merged**